packages/xstate-vue/test/UseMachine.vue

```diff
@@ -50,22 +50,13 @@ export default {
     const onFetch = () =>
       new Promise(res => setTimeout(() => res('some data'), 50));
 
-    const { current, send, service } = useMachine(fetchMachine, {
+    const { state, send, service } = useMachine(fetchMachine, {
       services: {
         fetchData: onFetch
       },
       state: persistedState
     });
-    watch(() =>
```

Comment:
i think this one was unused, `load` is not referenced anywhere as far as I can tell